package.json

```
@@ -42,7 +42,7 @@
   "devDependencies": {
     "@types/console-log-level": "1.4.2",
     "@types/node": "18.14.0",
-    "tempy": "1.0.1",
+    "tempy": "git+ssh://git@github.com:OpenINF/openinf-tempy.git#semver:^1.0.1",
```

```suggestion
    "tempy": "OpenINF/openinf-tempy.git#semver:^1.0.1",
```

https://yarnpkg.com/features/protocols
https://pnpm.io/cli/add#install-from-git-repository

---

omitting the leading protocol(s) seems fine in npm too, but hopefully, the trailing semver works

Refs: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#github-urls

feel free to approve and land